### PR TITLE
Add tests for BaselineFacade

### PR DESCRIPTION
**Pull request by schalkms**

### CODE REVIEW

File: detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/baseline/BaselineFacadeSpec.kt [hunk 1]
```diff
@@ -15,31 +15,44 @@ import java.nio.file.Path
...
     }
 })
 
+private fun assertCreate(fullPath: Path) {
+    val baselineFacade = BaselineFacade(fullPath)
+    baselineFacade.create(emptyList())
+    Files.newInputStream(fullPath).use<InputStream?, ListAssert<String>?> {
+        val reader = BufferedReader(InputStreamReader(it))
+        assertThat(reader.lines()).isNotEmpty
+    }
```
Comment (BraisGabin) [inline detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/baseline/BaselineFacadeSpec.kt]: What are you asserting here?

File: detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/baseline/BaselineFacadeSpec.kt [hunk 2]
```diff
@@ -15,31 +15,44 @@ import java.nio.file.Path
...
 
-        it("filterWithoutExistingBaseline") {
+        it("filters with an existing baseline file") {
             val path = Paths.get(resource("/smell-baseline.xml"))
             assertFilter(path)
         }
     }
 })
 
+private fun assertCreate(fullPath: Path) {
```
Comment (arturbosch) [inline detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/baseline/BaselineFacadeSpec.kt]: Nit: `assertCreated` `assertNonEmptyBaseline`

File: detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/baseline/BaselineFacadeSpec.kt (hunk 1 above)
Comment (schalkms) [reply detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/baseline/BaselineFacadeSpec.kt]: I assert that a baseline was created. I renamed it according to Artur's suggestion.

Comment (BraisGabin) [reply detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/baseline/BaselineFacadeSpec.kt]: With the name change I'm ok. But probably using some of this functions will make the code easier to read: http://joel-costigliola.github.io/assertj/core/api/org/assertj/core/util/Files.html

Comment (schalkms) [reply detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/baseline/BaselineFacadeSpec.kt]: Good point. Thanks!

**Pull request merged**